lib/tessel/wifi.js

```diff
@@ -38,21 +38,20 @@ Tessel.prototype.findAvailableNetworks = function() {
 };
 
 Tessel.prototype.getWifiInfo = function() {
-  var self = this;
-  return self.simpleExec(commands.getWifiInfo())
-    .then(function fetchedWiFiInfo(resultsJSON) {
+  return this.simpleExec(commands.getWifiInfo())
+    .then((resultsJSON) => {
```

Comment:
I kept them for the sake of consistency around using arrow functions for people new to the project and new to ES2015 features, since having only one parameter is the only case where we wouldn't use parens. Thanks for keeping me informed though. :)